src/heap.c

```diff
@@ -1,22 +1,22 @@
 #include "common.h"
 
-extern s32 D_80268000;
-extern s32 D_802FB800;
+extern HeapNode* heapCollisionHead; //D_80268000;
```

Comment:
We typically name globals with a prefix so they're distinguishable from other variables. so like `heap_collisionHead` for example. and those address comments can be removed

Reply:
Changed